flight_phase_files/HALO/EUREC4A_HALO_Flight-Segments_20200215.yaml

```
@@ -220,11 +219,11 @@ segments:
   - circling
   name: circling 1
   irregularities:
-  - 'circle 1 descent from FL320 to FL180'
-  - 'circle 1 lasts 68 minutes'
-  - 'circle 1 circle ranges over only 337 degrees'
-  - 'TTFS circle 2 starts 165 seconds prior to first dropsonde'
-  - 'circle 2 only 8 dropsondes'
+  - circle 1 descent from FL320 to FL180
+  - circle 1 lasts 68 minutes
+  - circle 1 circle ranges over only 337 degrees
+  - TTFS circle 2 starts 165 seconds prior to first dropsonde
+  - circle 2 only 8 dropsondes
```

This change does not affect the stored information and should be undone to clean up the diff:
```suggestion
  - 'circle 1 descent from FL320 to FL180'
  - 'circle 1 lasts 68 minutes'
  - 'circle 1 circle ranges over only 337 degrees'
  - 'TTFS circle 2 starts 165 seconds prior to first dropsonde'
  - 'circle 2 only 8 dropsondes'
```